GL/index.js

```diff
@@ -26,7 +26,7 @@ app.get('/getAllPeople', function(req, res) {
     data = axios.get('http://java-sample-api-2020.herokuapp.com/getAllPeople')
         .then((response) => {
             const data = response.data
```

Comment:
On line 28, the data variable is not used so consider removing that line.

Reply:
Agree with Jonathan